public/app/core/components/DynamicImports/ErrorLoadingChunk.tsx

```diff
@@ -0,0 +1,28 @@
+import React, { FunctionComponent } from 'react';
+import { Button } from '@grafana/ui';
```

Comment:
How about using `Forms.Button` ?

Reply:
Should we already start to mix the styles?

Reply:
I think in this case it's not an issue, as (i hope) error component are not that popular ;)